src/diffusers/loaders/ip_adapter.py

```diff
@@ -228,27 +229,57 @@ def load_ip_adapter(
         unet = getattr(self, self.unet_name) if not hasattr(self, "unet") else self.unet
         unet._load_ip_adapter_weights(state_dicts, low_cpu_mem_usage=low_cpu_mem_usage)
 
-    def set_ip_adapter_scale(self, scale):
+    def set_ip_adapter_scale(self, scale_configs: Union[float, Dict, List[Union[float, Dict]]], default_scale=0.0):
```

Comment:
let's try not to change arg names, so that `set_ip_adapter_scale(scale=...)` still works
```suggestion
    def set_ip_adapter_scale(self, scale: Union[float, Dict, List[Union[float, Dict]]], default_scale=0.0):
```

Reply:
Updated, thanks!